recipe/meta.yaml

```
@@ -1,80 +1,133 @@
 
+# break CI jobs per python version
 requirements:
-  
-  build:
-    - cmake
-    - make
-    - {{ compiler('c') }}
-    - {{ stdlib("c") }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
-
   host:
     - python
-    - packaging
-    - setuptools >=49.4.0
-    - pytorch >=2.1.0
-    - wheel
-    - pip
-    - libxcrypt
-  run:
-    - python
-    - pytorch
-    - torchvision
-    - transformers >=4.35.0
-    - tokenizers >=0.12.1
-    - accelerate
-    - sentencepiece
-    - lm_eval
-    - texttable
-    - toml
-    - attributedict
-    - protobuf
-    - torchvision
-    - tabulate
-    - cudatoolkit
-    - libtorch
 
-test:
-  imports:
-    - awq
-  # commands: # pending https://github.com/conda-forge/tcolorpy-feedstock/issues/12
-  #   - pip check
-  requires:
-    - pip
+outputs:
+  - name: autoawq_kernels
+    version: {{ kernel_version }}
+    build:
+      activate_in_script: true
+      script:
+        - sed -i 's/os.environ\["CC"\] = "g++"//' kernels/setup.py
+        - sed -i 's/os.environ\["CXX"\] = "g++"//' kernels/setup.py
+        - cd kernels
+        - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+      skip: true  # [cuda_compiler_version == "None"]
+      skip: true  # [osx or win]
+      skip: true  # [py == 38]
+      missing_dso_whitelist:
+        - '*/libtorch_python.so'
+
+    requirements:
+      build:
+        - cmake
+        - make
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}
+      host:
+        - python
+        - pip
+        - pytorch
+        {% if cuda_major >= 12 %}
+        - cuda-version {{ cuda_compiler_version }}
+        - cuda-cudart-dev
+        - libcublas-dev
+        - libcusolver-dev
+        - libcusparse-dev
+        {% endif %}
+    test:
+      imports:
+        - awq_ext
+
+{% if False %}
+  - name: autoawq
+    build:
+      script:
+        - export LDFLAGS="$(echo $LDFLAGS | sed 's/-Wl,--allow-shlib-undefined//g')"
+        - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+      skip: true  # [cuda_compiler_version == "None"]
+      skip: true  # [osx or win]
+      skip: true  # [py == 38]
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+      host:
+        - python
+        - {{ pin_subpackage('autoawq_kernels', exact=True) }}
+        - packaging
+        - setuptools >=49.4.0
+        - pytorch
+        - wheel
+        - pip
+        - requests
+        - libxcrypt
+      run:
+        - python
+        - {{ pin_subpackage('autoawq_kernels', exact=True) }}
+        - accelerate
+        - datasets
+        - tokenizers >=0.12.1
+        - transformers >=4.35.0
+        - typing_extensions >=4.8
+        - zstandard
+    test:
+      imports:
+        - awq
+      requires:
+        - pip
+      commands:
+        # pending https://github.com/conda-forge/tcolorpy-feedstock/issues/12
+        # - pip check
```

Should we have some line here 


```suggestion
      commands:
        # pending https://github.com/conda-forge/tcolorpy-feedstock/issues/12
        # - pip check
        - exit 0
```

Or do we want to comment the `commands` section

Should we have some line here 


```suggestion
      #commands:
      # pending https://github.com/conda-forge/tcolorpy-feedstock/issues/12
      #   - pip check
```

---

I don't get the point of your suggestion. The intent is clear and the linter doesn't complain - what do we gain by changing things?